content/kubermatic/master/tutorials-howtos/external-clusters/eks/_index.en.md

```diff
@@ -26,35 +43,67 @@ You should see the list of all available clusters in the region specified. Selec
 After the cluster is added, the KKP controller retrieves the cluster kubeconfig to display all necessary information.
 A healthy cluster has `Running` state. Otherwise, the cluster can be in the `Error` state. Move the mouse cursor over the state indicator to get more details.
 
```

Comment:
{{% notice info %}}
Creating External cluster doesn't create nodes by default but you can create by clicking "Add Machine Depkoyment"
{{% /notice %}}

Notes: as this by design in EKS